packages/util/src/useCombinedRefs.js

```diff
@@ -0,0 +1,17 @@
+import { useRef, useEffect } from 'react'
+
+export const useCombinedRefs = (...refs) => {
+  const targetRef = useRef()
+  useEffect(() => {
+    refs.forEach(ref => {
+      if (!ref) return
+      if (typeof ref === 'function') {
+        ref(targetRef.current)
```

Comment:
we probably don't need to support the older ref api. you can decided to leave it in or take it out

Reply:
Even though traditional class component callback refs may not be in use there is a new pattern using useCallback that is recommended for [measuring the dom for example ](https://reactjs.org/docs/hooks-faq.html#how-can-i-measure-a-dom-node). I'm going to leave this in place and add a test.